rules/rules.js

```diff
@@ -297,7 +298,8 @@ const getTriggeredData = function (input) {
         receivedCommand: null,
         receivedState: null,
         receivedTrigger: null,
-        itemName: evArr[0]
+        itemName: evArr[0],
+        module: input.get("module")
```

Comment:
This looks ok to me , although this will return a null value if not found, i think thats ok as long as we don't have any code that expects otherwise (like i'm assuming `oldState` and `newState` where we set to a empty string)   i'm not familiar with the module in the `input` , looks like its an ID/UUID to something , out of curiosity what are you using it for in rules ?

Reply:
> out of curiosity what are you using it for in rules

see above in the initial description.

for simple rules, I  implemented my own design pattern as described above, where I supply names for triggers, e.g. "BT1", "BT2", these labels I then use in a switch statement to execute code based on which trigger happened.

I also have some other rules (lights, shades, ..) where a single rule might have 100+ triggers built out of a JSON definition string. see: [https://community.openhab.org/t/tutorial-example-of-lights-control-using-lambdas-ported-to-ngre-python/68601?u=lukics](https://community.openhab.org/t/tutorial-example-of-lights-control-using-lambdas-ported-to-ngre-python/68601?u=lukics).

My rules are right now all written in Jython. I am preparing the ground to port these to JS.